iris-mpc-gpu/src/dot/kernel.cu

```diff
@@ -93,6 +93,67 @@ extern "C" __global__ void mergeDbResults(unsigned long long *matchResultsLeft,
     }
 }
 
+extern "C" __global__ void mergeDbResultsWithOrPolicyBitmap(unsigned long long *matchResultsLeft, unsigned long long *matchResultsRight, unsigned int *finalResults, size_t queryLength, size_t dbLength, unsigned int *matchCounter, unsigned int *allMatches, unsigned int *matchCounterLeft, unsigned int *matchCounterRight, unsigned int *partialResultsLeft, unsigned int *partialResultsRight, const unsigned long long *orPolicyBitmap) // 2D bitmap stored as 1D
+{
+
+    size_t rowStride64 = (dbLength + 63) / 64;
+
+    size_t totalBits   = dbLength * queryLength;
+    size_t numElements = (totalBits + 63) / 64; //  div_ceil
```

Comment:
@philsippl check if this is acceptable, was the only way I found to remove another argument from the function call